libbeat/autodiscover/providers/docker/docker.go

```diff
@@ -51,16 +51,16 @@ func AutodiscoverBuilder(bus bus.Bus, c *common.Config) (autodiscover.Provider,
 	var builders autodiscover.Builders
 	for _, bcfg := range config.Builders {
 		if builder, err := autodiscover.Registry.BuildBuilder(bcfg); err != nil {
-			logp.Warn("docker", "failed to construct autodiscover builder due to error: %v", err)
+			logp.Warn("failed to construct autodiscover builder due to error: %v", err)
```

Comment:
I know it was already there but I wonder if we should make this either error or info as we try not to use `Warn`. The problem with warn is that it's unclear if this is just an "Info" or needs action in which case it would be an error.

Reply:
i would like to keep those failure messages as just `Info` as they dont impact the coming up of the provider.